bxserum/provider/common.go

```diff
@@ -12,15 +12,15 @@ import (
 const (
 	MainnetSerumAPIHTTP = "https://virginia.solana.dex.blxrbdn.com"
 	MainnetSerumAPIWS   = "wss://virginia.solana.dex.blxrbdn.com/ws"
-	MainnetSerumAPIGRPC = "virginia.solana.dex.blxrbdn.com:9000"
+	MainnetSerumAPIGRPC = "virginia.solana.dex.blxrbdn.com:443"
 	TestnetSerumAPIHTTP = "http://serum-nlb-5a2c3912804344a3.elb.us-east-1.amazonaws.com/"
 	TestnetSerumAPIWS   = "ws://serum-nlb-5a2c3912804344a3.elb.us-east-1.amazonaws.com/ws"
-	TestnetSerumAPIGRPC = "serum-nlb-5a2c3912804344a3.elb.us-east-1.amazonaws.com:9000"
+	TestnetSerumAPIGRPC = "serum-nlb-5a2c3912804344a3.elb.us-east-1.amazonaws.com:443"
```

Comment:
From the other PR, it seems that everything but Mainnet has the port 80? https://github.com/bloXroute-Labs/serum-client-ts/pull/11/files

Reply:
ah, you're right. LB doesn't have ssl in test environments. fixed

Reply:
How does having SSL affect the port?

Reply:
default SSL port is 443 otherwise it's 80. In a web browser if you go to `http://serum-nlb-5a2c3912804344a3.elb.us-east-1.amazonaws.com/`, you're actually going to `http://serum-nlb-5a2c3912804344a3.elb.us-east-1.amazonaws.com:80`. But if you go to `https://...`, it's port 443.